svi/auto_thermal_reformer/identify_vars.py

```diff
@@ -0,0 +1,143 @@
+#  ___________________________________________________________________________
```

Comment:
This script has not been superseded by any of my (more) recent commits to this PR.